<launch>



    <node name="projectionSceneFirstProjector" pkg="projection_calibration" type="projection_calibration_node">
        <param name="configPath" value="src/projection_calibration/src/config1/config1.xml" />
        <param name="windowName" value="Projector 1" />
    </node>
    <node name="projectionSceneSecondProjector" pkg="projection_calibration" type="projection_calibration_node">
        <param name="configPath" type="str" value="src/projection_calibration/src/config2/config2.xml" />
        <param name="windowName" value="Projector 2" />
    </node>
    <!-- <node name="Projection Calibration 2" pkg="projection_calibration" type="projection_calibration_node">
    </node> -->
</launch>
